css/properties/shape-image-threshold.json

```diff
@@ -50,18 +50,17 @@
               "version_added": "24"
             },
             "safari": {
-              "prefix": "-webkit-",
-              "version_added": "7"
+              "version_added": "10.1"
             },
             "safari_ios": {
-              "version_added": null
+              "version_added": "10.3"
             },
             "samsunginternet_android": {
-              "version_added": null
+              "version_added": "4"
```

Comment:
Unlike the other browsers, `samsunginternet_android` version numbers do include the `.0` suffix, so they'll need to be added throughout (If you're curious, see #1784 for details)